tracim/tracim/lib/notifications.py

```diff
@@ -210,6 +211,24 @@ def _get_sender(self, user: User=None) -> str:
             email_address = email_address
         )
 
+    @staticmethod
+    def _log_notification(
+            recipient: str,
+            subject: str
+    ) -> None:
+        """Log notification metadata."""
+        from tracim.config.app_cfg import CFG
+        log_path = CFG.get_instance().EMAIL_NOTIFICATION_LOG_FILE_PATH
+        if log_path:
+            with open(log_path, 'a') as log_file:
```

Comment:
This method works but it is not efficient : file is open on each notification. Example: you update a document which will be notified to 100 users, the file will be open 100 times and closed 100 times.

Add a TODO in the code explaining this efficientless way of work.
(note: the way you fix the bug is enough for the moment)